core/trino-spi/src/test/java/io/trino/spi/TestSpiBackwardCompatibility.java

```diff
@@ -110,7 +111,10 @@ private static Set<String> getSpiEntities(ClassLoader classLoader, boolean inclu
             Class<?> clazz = classInfo.load();
             addClassEntities(entities, clazz, includeDeprecated);
         }
-        return entities.build();
+        return entities.build().stream()
+                // adding or removing final keyword is not considered as backward incompatible change
```

Comment:
```suggestion
                // Ignore `final` so that we can e.g. remove final from a SPI method.
                // While adding `final` can be a breaking change, we currently ignore such breakages.
```